src/thermal.rs

```diff
@@ -0,0 +1,40 @@
+use super::Error;
```

Comment:
The cooling devices didn't show fan speed on my laptop.

Reply:
hmmm, maybe its device or manufacture specific then

Reply:
> hmmm, maybe its device or manufacture specific then

Could be bios security. I'll check when I get a chance.

Reply:
So I couldn't find fan speed in the cooling device directory so I created its own issue since it might be more involved. @JakeRoggenbuck @Shuzhengz

Reply:
Okay, sounds good